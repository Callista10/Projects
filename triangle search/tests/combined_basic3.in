-N=6 -mindist=3 -rseed=8