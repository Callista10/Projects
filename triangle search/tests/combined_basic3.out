read 6 points
-37.37, 20.48
14.19, -4.09
-4.69, -0.50
This is a triangle
